android/app/build.gradle

```diff
@@ -165,73 +156,24 @@ android {
     }
     buildTypes {
         debug {
-            ndk {
-                abiFilters "arm64-v8a", "armeabi-v7a", "x86"
-            }
-            applicationIdSuffix ".debug"
             signingConfig signingConfigs.debug
+            resValue "string", "CodePushDeploymentKey", '""'
+            applicationIdSuffix ".debug"
         }
-
         release {
             // Caution! In production, you need to generate your own keystore file.
             // see https://facebook.github.io/react-native/docs/signed-apk-android.
-            ndk {
-                abiFilters "arm64-v8a", "armeabi-v7a"
-            }
             signingConfig signingConfigs.debug
```

Comment:
release and debug k liye separate signingConfig build karna hai. release k liye (farmcraft-release.keystore) and debug k liye(debug.keystore).